src/frontend/src/optimizer/plan_node/logical_scan.rs

```diff
@@ -86,6 +86,7 @@ impl LogicalScan {
             table_desc,
             indexes,
             predicate,
+            chunk_size: 1024,
```

Comment:
What about using an `Option` here? If it's `None`, then the batch executor will follow the value from the config, so that we don't need to hard code a `1024` here.

TIPS: wrap a primitive into a new message will make it `optional` in proto3 like this:
https://github.com/risingwavelabs/risingwave/blob/9e1ff935242f35f3a4acf7019f90a334e3c40eba/proto/stream_plan.proto#L632-L634